.gitignore

```diff
@@ -0,0 +1,59 @@
+# Byte-compiled / optimized / DLL files
```

Comment:
This file is from https://github.com/github/gitignore/blob/master/Python.gitignore